src/AD_ForwardDiff.jl

```diff
@@ -4,6 +4,8 @@
 
 import .ForwardDiff
 
+import DiffResults # should load DiffResults_helpers.jl
+
```

Comment:
I wonder if this is needed? ForwardDiff already loads DiffResults, so shouldn't the helpers be loaded automatically?

Reply:
I am wondering myself, but technically ForwardDiff *could* remove DiffResults as a dependency (make it conditional etc) and then we would have a problem.

My impression is that this is innocuous, so I would leave it in.

Reply:
True, it seems a bit safer to keep it.

Reply:
However, does this actually work? DiffResults is not a direct dependency, so wouldn't we need something like
```suggestion
import ForwardDiff.DiffResults # should load DiffResults_helpers.jl

```
?

Reply:
Ah no, I see ForwardDiff exports it:
https://github.com/JuliaDiff/ForwardDiff.jl/blob/6a6443b754b0fcfb4d671c9a3d01776df801f498/src/ForwardDiff.jl#L28

Reply:
Ah but again, we don't load ForwardDiff but just import it, so I guess we should still use
```suggestion
import ForwardDiff.DiffResults # should load DiffResults_helpers.jl

```
or something similar